build.sbt

```diff
@@ -80,8 +79,8 @@ lazy val commonSettings =
 
     // MUnit
     libraryDependencies ++= Seq(
-      "org.typelevel"     %% "scalacheck-effect-munit" % "1.0.1"  % Test,
-      "org.typelevel"     %% "munit-cats-effect-2"     % "1.0.2" % Test,
+      "org.typelevel"     %% "scalacheck-effect-munit" % "1.0.2"  % Test,
+      "org.typelevel"     %% "munit-cats-effect-2"     % "1.0.3" % Test,
```

Comment:
@tpolecat should this be munit-cats-effect-3 instead? :thinking:

Reply:
Yes, curious that it works at all.

Reply:
We're good to go if you can fix this.